actions/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
See https://github.com/learn-co-curriculum/react-type-ahead-wikipedia-search-lab/pull/2#issuecomment-251322148 for an idea on how to approach this instead of using `jsonp`